Fix visible variable issue

xyingsoft opened the pull request:
https://github.com/davidhemphill/presenter/issues/50

davidhemphill commented:
This sort of change would require a failing test to demonstrate the issue, or a test verifying the issue is fixed with your change.

xyingsoft commented:
I changed `test/Fixtures/HiddenAndVisibleAttributesPresenter`'s visible variable like following
```
public $visible = ['email'];
```
But test spec is passed, its critical.

Original code returns visible columns by index instead of column name. and since `name` is model keys first index, name is always returning and spec is passing whatever `$visible` variable is.

@davidhemphill I added spec to ensure my changes. Please review.

xyingsoft commented:
@davidhemphill Would you please review this PR?
